jsonrpc/src/content/results.json

```diff
@@ -154,6 +154,26 @@
       }
     }
   },
+  "RecursiveFindContentResult": {
+    "name": "recursiveFindContentResult",
+    "description": "One of either the data corresponding to the lookup target or a list of ENR records closer to the lookup target.",
```

Comment:
I would not let it return ENRs when it doesn't find the content, rather just an error.
Not much point in returning ENRs, that's rather for the non recursive call.